infoschema/tables.go

```diff
@@ -1662,18 +1662,26 @@ func GetStoreServerInfo(ctx sessionctx.Context) ([]ServerInfo, error) {
 		} else {
 			tp = tikv.GetStoreTypeByMeta(store).Name()
 		}
+
 		servers = append(servers, ServerInfo{
 			ServerType:     tp,
 			Address:        store.Address,
 			StatusAddr:     store.StatusAddress,
-			Version:        store.Version,
+			Version:        FormatTiflashVersion(store.Version),
 			GitHash:        store.GitHash,
 			StartTimestamp: store.StartTimestamp,
 		})
 	}
 	return servers, nil
 }
 
+func FormatTiflashVersion(version string) string {
```

Comment:
It is actually `FormatTiKVAndTiFlashVersion`, right? BTW, please add a comment for the exported method.

Reply:
> It is actually FormatTiKVAndTiFlashVersion, right

Good catch, Yes.

> please add a comment for the exported method.

thanks for your advice

Reply:
> thanks for your advice

It actually will the CI test:

```
[2021-06-17T03:30:58.992Z] GO111MODULE=on go build -o ../bin/revive github.com/mgechev/revive

[2021-06-17T03:30:58.992Z] linting

[2021-06-17T03:30:58.992Z]   ✘  https://revive.run/r#exported  exported function FormatTiflashVersion should have comment or be unexported  

[2021-06-17T03:30:58.992Z]   infoschema/tables.go:1678:1

[2021-06-17T03:30:58.992Z] 

[2021-06-17T03:30:58.992Z] ✘ 1 problem (1 error, 0 warnings)

[2021-06-17T03:30:58.992Z] 

[2021-06-17T03:30:58.992Z] Errors:

[2021-06-17T03:30:58.992Z]   1  exported  
```